AutoSteamApp/Process Memory/SaveData.cs

```diff
@@ -0,0 +1,176 @@
+﻿using AutoSteamApp.Configuration;
+using AutoSteamApp.Helpers;
+using Logging;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace AutoSteamApp.ProcessMemory
+{
+
+    public class SaveData
+    {
+
+        #region Fields
+
+        /// <summary>
+        /// The Monster Hunter World: Iceborne process to load player data from.
+        /// </summary>
+        Process MHWProcess;
+
+        /// <summary>
+        /// The address of the steamworks save data.
+        /// </summary>
+        ulong SteamworksSaveDataAddress;
```

Comment:
A better name for this would be `SaveDataAddress` as this is actually the START of the `Save` information in memory.